samples/msal-node-samples/ElectronSystemBrowserTestApp/public/errorTemplate.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self'" />
+    <title>Electron system browser auth redirect page</title>
+    <style>
+    .container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+         padding: 10px;
```

Comment:
super nit: indentation is off on this line

Reply:
Done!